### ISSUE TITLE
Color Accessibility

### ISSUE THREAD

**Issue by virtual** [virtual/fcc-recipe-box#6]
Some white font colors are marked inaccessible though they are pulling the wrong parent background color on check.

The foreground color `#5ed5cb` should be changed to `#76dcd3` to pass WCAG 2.0 AA.
